larq/optimizers.py

```diff
@@ -1,9 +1,9 @@
 from larq import utils
 
 if utils.tf_1_14_or_newer():
-    from larq.optimizers_v2 import Bop
+    from larq.optimizers_v2 import Bop, CaseOptimizer
 
-    __all__ = ["Bop"]
+    __all__ = ["Bop", "OptimizerGroup"]
```

Comment:
This should be, or we need to rename `CaseOptimizer`
```suggestion
    __all__ = ["Bop", "CaseOptimizer"]
```

Reply:
Yeah oops haha, already fixed in 1b49c96 but hadn't pushed yet.